server/java/src/main/java/com/godaddy/asherah/grpc/ServerApp.java

```diff
@@ -26,37 +26,71 @@
     }
   }
 
+  @CommandLine.ArgGroup
+  private DynamoDbFlags dynamoDbFlags;
+
+  static class DynamoDbFlags {
+    @CommandLine.ArgGroup(exclusive = false)
+    private EndPoint endPoint;
+
+    static class EndPoint {
+      @CommandLine.Option(names = "--dynamodb-endpoint", required = true,
+          defaultValue = "${env:ASHERAH_DYNAMODB_ENDPOINT}",
+          description = "The DynamoDb service endpoint (only supported by DYNAMODB)")
+      private static String endpoint;
+      @CommandLine.Option(names = "--dynamodb-signing-region", required = true,
+          defaultValue = "${env:ASHERAH_DYNAMODB_SIGNING_REGION}",
+          description = "The DynamoDb region to use for SigV4 signing of requests (only supported by DYNAMODB)")
+      private static String signingRegion;
+    }
+
+    @CommandLine.ArgGroup
+    private Region region;
+
+    static class Region {
+      @CommandLine.Option(names = "--dynamodb-region", defaultValue = "${env:ASHERAH_DYNAMODB_REGION}",
+          description = "The AWS region for DynamoDB requests (only supported by DYNAMODB)")
+      private static String region;
+    }
+  }
+
   // Options to configure the metastore
   @CommandLine.Option(names = "--metastore-type", defaultValue = "${env:ASHERAH_METASTORE_MODE}",
       completionCandidates = MetastoreTypes.class,
       description = "Type of metastore to use. Possible values: ${COMPLETION-CANDIDATES}", required = true)
-  private static String metastoreType;
+  private String metastoreType;
   @CommandLine.Option(names = "--jdbc-url", defaultValue = "${env:ASHERAH_CONNECTION_STRING}",
       description = "JDBC URL to use for JDBC metastore. Required for JDBC metastore.")
-  private static String jdbcUrl;
+  private String jdbcUrl;
+  @CommandLine.Option(names = "--enable-key-suffix", defaultValue = "${env:ASHERAH_ENABLE_KEY_SUFFIX}",
```

Comment:
I think that this flag should be renamed to `key-suffix`.
The word `enable` should be used more for boolean values that don't require a parameter values. Example `--enable-session-caching`